tests/behavior/behavior.rs

```diff
@@ -1081,7 +1082,7 @@ async fn test_presign_write(op: Operator) -> Result<()> {
     req.headers_mut()
         .insert(header::CONTENT_LENGTH, content.len().to_string().parse()?);
 
-    let client = hyper::Client::builder().build(hyper_tls::HttpsConnector::new());
+    let client = HttpClient::new();
```

Comment:
How about always using `hyper_tls` for tests? So that we don't need to expose `HttpClient`.

We can specify `hyper_tls` in `dev-dependencies`.

Reply:
No problem. The idea to use `HttpClient` here was to test both openssl and rustls implementation. But tests from `http_body.rs` seems sufficient for now.